### ISSUE TITLE
toggleInlineStyle doesn't toggle more than once on collapsed selection

### ISSUE THREAD

**Issue by sPaCeMoNk3yIam** [facebook/draft-js#1832]
BUG

**What is the current behavior?**
If `toggleInlineStyle` is invoked on an empty selection, the specified style is toggled in relation to the previous style. If it then is invoked another time, it doesn't toggle back to the style at the beginning.

state A -[toggleInlineStyle is called]-> state !A -[toggleInlineStyle is called]-> state !A ...

**What is the expected behavior?**
state A -[toggleInlineStyle is called]-> state !A -[toggleInlineStyle is called]-> state A

**Which versions of Draft.js, and which browser / OS are affected by this issue? Did this work in previous versions of Draft.js?**
It's the case for all browsers and I use v0.10.5 (latest).

**Comment by niveditc** [comment]
Good catch! I think that this is probably worth fixing. PRs are welcome for this issue!

**Comment by niveditc** [comment]
I was revisiting this issue & can't seem to repro it. Can you please provide an example of this behavior?
